src/com/xilinx/rapidwright/timing/DelayModelBuilder.java

```diff
@@ -108,24 +108,66 @@ private static void newDelayModel(String mode, String source, String fileName) {
     /**
      * For unit testing.
      */
-    private static int testLogicDelay(DelayModel a, String belName, List<String> config, 
-            String[] src, String[] dst) {
-
+//    private static int testLogicDelay(DelayModel a, String belName, List<String> config, 
```

Comment:
Let's remove this method or uncomment it.  It won't get maintained as a comment.